.github/workflows/publish.yaml

```diff
@@ -27,5 +27,5 @@ jobs:
           HF_TOKEN: ${{ secrets.HF_TOKEN_PUSH }}
           HF_REPO: ${{ secrets.HF_REPO }}
         run: |
-          python quartz_solar_forecast/utils/hf_upload.py
+          python scripts/hf_upload.py
```

Comment:
How come this had to move?

Reply:
Something about it sitting in the library with `__main__` didnt sit right, and we couldnt test the whole script. 
Hope that is ok?

Reply:
Yep all good